pkg/aws/rds.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 The Kanister Authors.
```

Comment:
I think we should organize the `aws` pkg - move `rds`, `ec2`, `config` into sub-packages.

Also `rds.NewClient()` works better than `aws.NewRDSClient()`

Reply:
Makes sense. Will do the changes